colossalai/inference/config.py

```diff
@@ -140,3 +141,18 @@ def to_generation_config(self, model_config) -> GenerationConfig:
                 meta_config[type] = getattr(model_config, type)
 
         return GenerationConfig.from_dict(meta_config)
+
+    @classmethod
+    def from_cli_args(cls, args: argparse.Namespace) -> "InferenceConfig":
```

Comment:
Consider make it consistent with the API shown in line 143, i.e. use `from_dict` instead. An example can be 

```python
args_dict = vars(args)
config = InferenceConfig.from_dict(args_dict)
```